GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -31,30 +46,101 @@ protected override string VerbName
 
         public override void Execute()
         {
-            this.localConfig = new LocalGVFSConfig();
+            if (GVFSPlatform.Instance.IsUnderConstruction)
+            {
+                this.ReportErrorAndExit("`gvfs config` is not yet implemented on this operating system.");
+            }
 
+            this.localConfig = new LocalGVFSConfig();
             string error = null;
-            bool isRead = string.IsNullOrEmpty(this.Value);
 
-            if (isRead)
+            if (this.IsMutuallyExclusiveOptionsSet(out error))

```

Comment:
Testing note: this is no longer catching the case where no params were specified. Running just `gvfs config` now produces an empty result. It should either show the same as -l or show an error, and it seems that showing an error would be more consistent with git's behavior.

Reply:
Missed it, Thank you! 

Updated screenshot-

<img width="467" alt="screen shot 2018-10-18 at 5 11 24 pm" src="https://user-images.githubusercontent.com/378580/47184560-01948380-d2f9-11e8-907d-7fbcfcfd2c56.png">

Reply:
`git config` will print out the options for the command.  I believe most commands in git will do this when it doesn't know what to do with the command.

Reply:
And that would be fine behavior if there's a method we can call to have the CommandLineParser print out the help for the current command. If that is not an option, then I don't think we should duplicate the help text, and telling the user to run the help is adequate.